invenio_app_ils/config.py

```diff
@@ -148,6 +148,30 @@ def _(x):
 #: Default time zone
 BABEL_DEFAULT_TIMEZONE = "Europe/Zurich"
 
+###############################################################################
+# Notifications configuration
+###############################################################################
+# The backends to use when sending notifications.
+ILS_NOTIFICATIONS_BACKENDS_BUILDER = "invenio_app_ils.notifications.backends:notifications_backend_builder"
+# Notification message creator
+ILS_NOTIFICATIONS_MSG_BUILDER = "invenio_app_ils.notifications.messages:notification_msg_builder"  # noqa
+# Override default global common templates
+ILS_NOTIFICATIONS_TEMPLATES = {"footer": "footer.html"}
+# CIRCULATION
+# Notification message creator for loan notifications
+ILS_NOTIFICATIONS_MSG_BUILDER_CIRCULATION = "invenio_app_ils.circulation.notifications.messages:notification_loan_msg_builder"  # noqa
```

Comment:
circulation configs should be probably moved to `invenio_app_ils/circulation/config.py`